src/status_im/communities/core.cljs

```diff
@@ -307,6 +340,15 @@
                                    (log/error "failed to fetch communities" %)
                                    (re-frame/dispatch [::failed-to-fetch %]))}]})
 
+(rf/defn fetch-contract-communities
```

Comment:
new functionality should be implemented in status-im2 namespace

Reply:
> ok so its something similar to what we do in `<-rpc` methods, so probably if we implement this, it should be general solution, but we should discuss with the team

Yes and thank you @flexsurfer! Yes, it effectively replaces the key-by-key renaming of the `<-rpc` family of functions. I observed that some keys have been overlooked (not converted), so we currently have an (undesirable IMO) mix of camelcase and kebabcase (in communities, at least). So instead of hunting down the unconverted keys, I looked into a general solution (which I commonly see in previous js<->clj codebases). I timed the two approaches and they were similar in computation time (in the order of 10's of ms)